app/segments/drill/[id]/submission/index.js

```diff
@@ -8,12 +8,37 @@ import Result from "./result";
 
 import ErrorComponent from "~/components/errorComponent";
 import { useDrillInfo } from "~/hooks/useDrillInfo";
+import DialogComponent from "../../../../../components/dialog";
```

Comment:
Do we want to do the "~/components/dialog" here?

Reply:
Yes